src/container/container_iv.c

```diff
@@ -1071,6 +1075,33 @@ cont_iv_ec_agg_eph_refresh(void *ns, uuid_t cont_uuid, daos_epoch_t eph)
 						  IV_CONT_AGG_EPOCH_BOUNDRY);
 }
 
+int
+ds_cont_fetch_ec_agg_boundary(void *ns, uuid_t cont_uuid)
+{
+
+	struct cont_iv_entry	iv_entry = { 0 };
+	int			rc;
+
+	/* Only happens on xstream 0 */
+	D_ASSERT(dss_get_module_info()->dmi_xs_id == 0);
+	uuid_copy(iv_entry.cont_uuid, cont_uuid);
+	rc = crt_group_rank(NULL, &iv_entry.iv_agg_eph.rank);
```

Comment:
seems the iv_agg_eph.rank not really needed for IV_CONT_AGG_EPOCH_BOUNDRY, so may need not get rank here.